src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -709,6 +713,8 @@ case class JesBackend(actorSystem: ActorSystem)
   }
 
   override def findResumableExecutions(id: WorkflowId)(implicit ec: ExecutionContext): Future[Map[ExecutionDatabaseKey, JobKey]] = {
-    globalDataAccess.findResumableJesExecutions(id)
+    globalDataAccess.findResumableExecutions(id, IsResumable, BuildJobKey)
   }
+
+  override def executionInfoKeys: Seq[String] = Seq("JES_RUN_ID", "JES_STATUS")
```

Comment:
Seq() creates a List, it isn't a generic entity. Calling it with Seq obscures that fact and can lead to bugs (in fact that has already happened in the lifespan of Cromwell). It's admittedly a minor thing but it makes it more difficult to reason about what you have and for the cost of an extra character you're saying explicitly what you're doing. 

As an example I recently asked @Horneth to de-Seq something w/ a List and he pointed out that it really should be a Set. That was a positive example IMO since having to think about the right collection type led to a better concrete instantiation.